virt/virt-4-10-release-notes.adoc

```diff
@@ -196,44 +194,69 @@ $ oc annotate --overwrite -n openshift-cnv hyperconverged kubevirt-hyperconverge
 ----
 <1> Replace `<cpu_model>` with the actual CPU model value. You can determine this value by running `oc describe node <node>` for all nodes and looking at the `cpu-model-<name>` labels. Select the CPU model that is present on all of your nodes.
 
-* If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887140[*BZ#1887140*])
+* If you use Red Hat Ceph Storage or Red Hat OpenShift Data Foundation Storage, cloning more than 100 VMs at once might fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1989527[*BZ#1989527*])
+** As a workaround, you can perform a host-assisted copy by setting `spec.cloneStrategy: copy` in the storage profile manifest. For example:
 +
-** To unlock the account, log in to the Manager and enter the following command:
-+
-[source,terminal]
+[source,yaml]
 ----
-$ ovirt-aaa-jdbc-tool user unlock admin
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+  name: <provisioner_class>
+#   ...
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Filesystem
+  cloneStrategy: copy <1>
+status:
+  provisioner: <provisioner>
+  storageClass: <provisioner_class>
 ----
+<1> The default cloning method set as `copy`.
 
-* If you run {VirtProductName} 2.6.5 with {product-title} 4.8 or later, various issues occur. You can avoid these issues by upgrading {VirtProductName} to version 4.8 or later.
-+
-** In the web console, if you navigate to the *Virtualization* page and select *Create* -> *With YAML* the following error message is displayed:
-+
-[source,text]
-----
-The server doesn't have a resource type "kind: VirtualMachine, apiVersion: kubevirt.io/v1"
-----
-+
-*** As a workaround, edit the `VirtualMachine` manifest so the `apiVersion` is `kubevirt.io/v1alpha3`. For example:
+* In some instances, multiple virtual machines can mount the same PVC in read-write mode, which might result in data corruption. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1992753[*BZ#1992753*]))
+** As a workaround, avoid using a single PVC in read-write mode with multiple VMs
+
+* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
+** As a workaround, see xref:../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[Silencing alerts].
```

Comment:
Please change this to `** As a workaround, xref:../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence the alert].`